src/utilities/utility.ts

```diff
@@ -105,6 +105,24 @@ export async function getWorkflowFilePath(githubToken: string): Promise<string>
     return Promise.resolve(workflowFilePath);
 }
 
+export async function getRepositoryId(githubToken: string): Promise<number> {

```

Comment:
Set the return type to `number`.
This way your function would return a `Promise<number>` instead of `Promise<Promise<number>>` because of the async wrapping

Reply:
I think it wraps only if the Promise is missing. I would like to be consistent with rest of the code.